src/microbe_stage/particles/CellBurstEffect.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Godot;
+
+public class CellBurstEffect : Particles, ITimedLife
+{
+    public Microbe Host;
```

Comment:
Why's this public. Also does this work with saving? It would be pretty bad if this reference is used after the microbe scene is disposed, or if this is re-loaded from a save as that would cause a resource leak (I think, I'm not 100% sure).

Reply:
Need to set it from the microbe script. Also no I forgot to make this saveable, I'll try replicating the json setup in AgentProjectile.cs here, assuming it'll work the same.

Reply:
As this just uses the Radius from the Host. It would be 10 times better to just have a Radius property in this class.

Reply:
Also, yeah I think referencing the entire microbe is a bit overkill, will change it to just a float for the cell's radius.